### PR TITLE
Replace some paper-items by list-item

### PR DESCRIPTION
**Pull request by silamon**
## Proposed change
Replace some paper-items and wrap them in a list with list-items. 

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/cloud/account/cloud-account.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import "@material/mwc-button";
 import "@material/mwc-list/mwc-list-item";
```
Comment (bramkragten) [inline src/panels/config/cloud/account/cloud-account.ts:2]: This should be removed as `ha-list-item` is (should be?) used

File: src/panels/config/cloud/account/cloud-account.ts [hunk 2]
```diff
@@ -103,24 +105,24 @@ export class CloudAccount extends SubscribeMixin(LitElement) {
...
-                    "ui.panel.config.cloud.account.connection_status"
-                  )}
-                </paper-item-body>
-                <div class="status">
-                  ${this.cloudStatus.cloud === "connected"
-                    ? this.hass.localize(
-                        "ui.panel.config.cloud.account.connected"
-                      )
-                    : this.cloudStatus.cloud === "disconnected"
+                <mwc-list>
```
Comment (bramkragten) [inline src/panels/config/cloud/account/cloud-account.ts:108]: No need to use `mwc-list` here, as it is just 1 item?

File: src/panels/config/cloud/account/cloud-account.ts [hunk 3]
```diff
@@ -65,30 +65,32 @@ export class CloudAccount extends SubscribeMixin(LitElement) {
...
-                                this.hass.config
-                              )
-                            : ""
-                        )
-                      : this.hass.localize(
-                          "ui.panel.config.cloud.account.fetching_subscription"
-                        )}
-                  </div>
-                </paper-item-body>
+                <mwc-list>
```
Comment (bramkragten) [inline src/panels/config/cloud/account/cloud-account.ts:68]: No need for `mwc-list`

**Comment by bramkragten** [comment]
Removing `paper-item-icon-color` is gonna piss off some people as themes will break, but it is the correct thing to do.

**Comment by silamon** [comment]
> Removing `paper-item-icon-color` is gonna piss off some people as themes will break, but it is the correct thing to do.

I've reverted this change for now. When we get closer to removing paper-items and paper-list, we can do a fallback.

**Comment by bramkragten** [comment]
> I'm also wrapping list-items into a list, even though there is only one item in it. paper-inputs were flexible enough, mwc-list-items specify list in the tag and should be used in a list I guess.

`mwc-list` only adds features as setting a selected item, and going through the list with your keyboard, since it isn't used as a list here, but just as a layout, we should not use `mwc-list`.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**